docs/exchange-operators/rosetta/send-requests.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Sending requests
+description: Learn about Mina-specific request/response models
+---
```

Comment:
You can add the metadata frontmatter with a future update https://github.com/o1-labs/docs2/wiki/SEO-metadata-front-matter